<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>job_tab</class>
 <widget class="QWidget" name="job_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>161</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="job_tab_layout" columnstretch="3,0,1">
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="folders_grp">
     <property name="title">
      <string>Folders</string>
     </property>
     <layout class="QGridLayout" name="folders_grp_layout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="base_folder_label">
        <property name="text">
         <string>Base:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="base_folder_edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="base_folder_set_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Edit-16.png</normaloff>:/icons/icons/icons8-Edit-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="base_folder_open_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Open-16.png</normaloff>:/icons/icons/icons8-Open-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="drone_folder_label">
        <property name="text">
         <string>Drone:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="drone_folder_edit"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="drone_folder_set_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Edit-16.png</normaloff>:/icons/icons/icons8-Edit-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="drone_folder_open_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Open-16.png</normaloff>:/icons/icons/icons8-Open-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="scans_folder_label">
        <property name="text">
         <string>Scans:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="scans_folder_edit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="scans_folder_set_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Edit-16.png</normaloff>:/icons/icons/icons8-Edit-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="scans_folder_open_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Open-16.png</normaloff>:/icons/icons/icons8-Open-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="assets_folder_label">
        <property name="text">
         <string>Assets:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="assets_folder_edit"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="assets_folder_set_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Edit-16.png</normaloff>:/icons/icons/icons8-Edit-16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="assets_folder_open_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Open-16.png</normaloff>:/icons/icons/icons8-Open-16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="scene_grp">
     <property name="title">
      <string>Scene</string>
     </property>
     <layout class="QHBoxLayout" name="scene_grp_layout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="scene_site_btn">
        <property name="toolTip">
         <string>Site</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Road-30.png</normaloff>:/icons/icons/icons8-Road-30.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scene_vehicle_btn">
        <property name="toolTip">
         <string>Vehicle</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Crashed Car-30.png</normaloff>:/icons/icons/icons8-Crashed Car-30.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="tools_grp">
     <property name="title">
      <string>Tools</string>
     </property>
     <layout class="QVBoxLayout" name="tools_grp_layout">
      <item>
       <widget class="QPushButton" name="google_maps_btn">
        <property name="text">
         <string>Google Maps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="google_earth_btn">
        <property name="text">
         <string>Google Earth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openair_btn">
        <property name="text">
         <string>OpenAir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="pix4d_grp">
     <property name="title">
      <string>Pix4D</string>
     </property>
     <layout class="QHBoxLayout" name="pix4d_grp_layout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="pix4d_site_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Site</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Road-30.png</normaloff>:/icons/icons/icons8-Road-30.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pix4d_vehicle_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Vehicle</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/icons8-Crashed Car-30.png</normaloff>:/icons/icons/icons8-Crashed Car-30.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
